foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -19,7 +19,6 @@ Don't forget to commit early & often! You can [reference the Commit Message less
 3.  Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'.  We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
 4.  Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
     *   Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
-    *   Account for TIES by re-playing the round.
```

Comment:
I'm on the fence about just removing this. I feel like if we don't explicitly say what to do about ties we'll get _lots_ of questions about it.

Reply:
Perhaps just a small change here:

> and then return a string that declares the winner or tie of the round like so: `"You Lose! Paper beats Rock"`

Reply:
Yeah that might be enough.

Reply:
@wise-king-sullyman can you please review my addition when you have a chance.  Thank you !